ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -490,4 +517,7 @@ public DDLCommandResult executeDdlStatement(String sqlExpression, final DDLState
     return queryEngine.handleDdlStatement(sqlExpression, statement, streamsProperties);
   }
 
+  public static SchemaRegistryClient getSchemaRegistryClient() {
```

Comment:
We shouldn't need this to be static. I think the only reason it is is for `buildStream(...)` in which case we should update `buildStream(...)` to have a `SchemaRegistryClient` param. Moving forward (in a future PR) we should change the `buildStream(...)` method to pass in an object that is a container for the things we need, i.e, kafkaTopicClient, schemaRegistery etc